src/controller/FileBrowserController.vala

```diff
@@ -28,7 +28,7 @@ namespace BrickManager {
     public class FileBrowserController : Object, IBrickManagerModule {
         const string PARENT_DIRECTORY_TEXT = "../";
         const string INITAL_DIRECTORY = "/home/robot";
-        const string USER_NAME = "ev3dev";
+        const string USER_NAME = "robot";
```

Comment:
How did I miss this? :smiley: 

Although now I am wondering if we still need this if `robot` is a member of the `tty` group. Do you remember if `openvt` sets permissions of `/dev/ttyX` to `root:root`?